### PR TITLE
Add oVirt ovirt_quotas and ovirt_quotas_facts modules

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

- New Module Pull Request
##### COMPONENT NAME

- ovirt_quotas
- ovirt_quotas_facts
##### SUMMARY

This PR adds two new modules to manage oVirt quotas.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @machacekondra. This module is going into community review.

We notice that you are one of the maintainers (@machacekondra) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by machacekondra** [comment]
@mwperina @ryansb

**Review by mwperina — APPROVED** [review]
Looks good to me, thanks! :+1:

**Comment by machacekondra** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @machacekondra. Since you are one of the maintainers (@machacekondra) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
Thanks again to @machacekondra. This module is going into community review.

We notice that you are one of the maintainers (@machacekondra) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/ovirt/ovirt_quotas_facts.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        module.exit_json(
+            changed=False,
+            ansible_facts=dict(
+                ovirt_quotas=[
+                    get_dict_of_struct(c) for c in quotas
+                ],
+            ),
+        )
+    except Exception as e:
+        module.fail_json(msg=str(e))
```
Comment (ryansb) [inline cloud/ovirt/ovirt_quotas_facts.py]: Same here, please surface the full exception.

**Comment by machacekondra** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @machacekondra. Since you are one of the maintainers (@machacekondra) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Review by mwperina — APPROVED** [review]
:+1:

**Pull request merged**